app/controllers/publish/courses/age_range_controller.rb

```diff
@@ -3,6 +3,8 @@ module Courses
     class AgeRangeController < PublishController
       include CourseBasicDetailConcern
       decorates_assigned :course
+      before_action :build_recruitment_cycle, only: %i[edit update]
+      before_action :build_provider, only: %i[edit update]
```

Comment:
```suggestion
```

It should be using https://github.com/DFE-Digital/teacher-training-api/blob/db1ca9b0b3a8c8d999e544cf1028766f1814b47b/app/controllers/publish/publish_controller.rb#L9-L17